### ISSUE TITLE
A&R -> Exported report for missing data - Issues with the Notes column

### ISSUE THREAD

**Issue by manuji** [bcgov/bcparks-ar-admin#412]
Scenario:
* Navigate to the A&R on TEST environment: https://test-ar.bcparks.ca/export-reports
* Navigate to the export report page and then select the Missing report type
* Export the report and check the column for notes. 

Issue:
* The Notes column duplicate the note from one activity to all the other activities even when that activity is not applicable to the park/facility. 
* The Notes column shows "Undefined" if the Notes field is empty. 
![Image](https://github.com/user-attachments/assets/5a612ec5-e668-4102-a699-aff0c8b66486)

Expected:
* The notes column should be re-evaluated to find the best way to display the needed information rather than duplicating the notes to every column. 
* Should only display notes from the correct activity. 
* If the note field is empty it would be ideal to leave the note on exported report to be empty as well. @sallyhaynes @davidclaveau feel free to correct me if you expect otherwise. ![Image]

**Comment by davidclaveau** [comment]
@manuji not sure how this slipped through the cracks, but this should be resolved in the next PR!

@nicolebottoms @sallyhaynes Manuji brings up a good point about the notes repeating. I think the logic for getting the `Notes` to appear for it's respective activity _might_ be a bit more work than it's worth, but would there be any issue with making one main header column for `Notes` so it gets printed and doesn't repeat? Example:

![Image](https://github.com/user-attachments/assets/6e381bc4-1ea0-4017-bc15-ca7254ac1b7e)

**Comment by davidclaveau** [comment]
@manuji this should be ready for review!

There should be a fix to the Notes section so it doesn't show `undefined` if there aren't any notes available. Also, I added a tweak so the Notes column just appears the one time at the start (after Bundle, Park, Subarea and Month).

Added a QoL fix on the frontend so the calendar input on the export page adjusts a bit better on different screen sizes, and the progress percentage bar for exports should have zero decimal places now 😄 no more _30.123785723495289%_  when exporting!

Let me know if anything looks amiss!
